lib/react_client/react_interop.dart

```diff
@@ -314,7 +323,6 @@ class ComponentStatics {
 @anonymous
 class JsComponentConfig {
   external factory JsComponentConfig({
-    Iterable<String> childContextKeys,
-    Iterable<String> contextKeys,
+    contextType,
```

Comment:
If `JsComponentConfig` is public API, it should merely be deprecated with a `@Deprecated('6.0.0')` annotation, and these changes should be rolled into a new `JsComponentConfig2` class.